### DISCUSSION TITLE
Change link color

### DISCUSSION THREAD

**Discussion by kyrathasoft** [inkle/ink#839] [Writing in ink]
I am trying to learn how to use Ink. The link color is orange-ish (gag). The hover link color is black, which is fine. How to change orange-ish link color in the style.css or elsewhere?

I was thinking the following element, but changing its value (for instance, to #ffff00 or 'blue') doesn't seem to make a difference:

a {
    font-weight: 700;
    color: #b97c2c;
    font-family: sans-serif;
    transition: color 0.6s;
    text-decoration: none;
}

a:hover {
    color: black;
    transition: color 0.1s;
}

**Comment by RTalaro** [comment]
This is 3 years later but for anyone that finds this question, the [tutorial](https://www.inklestudios.com/ink/web-tutorial/) hints at it slightly.
I don't know why it isn't working for you, but when I set the color in a, it changes the colors of all choices for me perfectly. I will add that if you want to change link text by class, you have to detail the class and then a afterwards like so:

```
// all choice links will appear black
a {
font-weight: 700;
color: black;
font-family: sans-serif;
transition: color 0.6s;
text-decoration: none;
}

// all choice links tagged with `# CLASS: orange` will appear orange
.orange a {
    color: coral;
}

// all paragraphs tagged with `# CLASS: orange` will appear orange
.orange {
    color: coral;
}
```
